spec/features/goal_tool_spec.rb

```diff
@@ -15,7 +15,7 @@
   end
 
   scenario "Displays due date" do
-    expect(page).to have_content "Due: #{Date.today.to_s(:date)}"
+    expect(page).to have_content "Due: #{Date.today.strftime("%b %d %Y")}"
```

Comment:
Well see I wanted a separation between the spec and how it is actually implementation.  I tried using a string; i.e., "Jan 3 2014" but that isn't going to work.  I thought maybe I could do `"#{Date.today.month + ...}"` but that seems like a pain in the butt.  So instead of using the code itself, I thought this was a good solution.  Any more thoughts or suggestions would be great.  Thanks!